.github/workflows/contract-monitor.yml

```diff
@@ -0,0 +1,56 @@
+name: Monitor Stellar Contract
+
+on:
+  schedule:
+    - cron: '0 8 * * 1' #every Monday at 8:00 AM
```

Comment:
For this PR, can you change the setting so that we can run this in the PR as well? In general we also need to see to be able to manually trigger like we have on other flows.